Source/WebCore/page/ImageOverlayController.h

```diff
@@ -47,6 +47,7 @@ class FloatQuad;
 class Page;
 class RenderElement;
 struct GapRects;
+class WeakPtrImplWithEventTargetData;
```

Comment:
I believe we sort alphabetically for all of `class` then all of `struct`.

Reply:
Looks like this is a case not covered by `check-webkit-style`, as it shows no errors with either style. I'll update the patch.